src/router.js

```diff
@@ -10,7 +11,8 @@ export default new Router({
   routes: [
     {
       path: routes.HOME,
-      redirect: { name: 'login' }
+      name: 'navBar',
+      component: NavBar
```

Comment:
this is very strange the navbar should be added to the protected routes not only to the home